### DISCUSSION TITLE
Using default installed browser

### DISCUSSION THREAD

**Discussion by douden** [SafeExamBrowser/seb-win-refactoring#380] [Q&A] [answered]
At my university we are trying to implement the use of SEB for taking on-campus digital exams. However our security software team has concerns about SEB having it's own build-in browser which does not seem to support (security) hotfixes.

Is there a way to make SEB use the standard browser installed on the end-users (i.e. students) OS?

With this I mean that if a student has Chrome installed as his/her default browser, SEB will use the Chrome browser (in kiosk mode).

Any information/reaction will be highly appreciated.

Regards, Dennis

**Comment by dbuechel** [comment, marked as answer]
We always update the integrated browser engine (Chromium) before making a new release, and we'd certainly release a new patch version in case there should be a high-risk vulnerability in the engine.

You can disable the integrated browser and add any standard browser as permitted application as described in the documentation, see https://safeexambrowser.org/windows/win_usermanual_en.html#BrowserPane for disabling the integrated browser resp. https://safeexambrowser.org/windows/win_usermanual_en.html#ApplicationsPane for adding a third-party application.

The integrated browser engine however provides a lot of additional features (like URL filtering, quit links etc.) which won't be available when using  another browser. Using a standard browser furthermore basically opens the entire internet and local file system access to a student, so that's most likely not what you'd want in an exam situation.

**Reply by douden** [reply]
Thank you for replying so quickly.

Your assurances make me happy, as with those I can go back to the security team.

I do agree that using another browser makes the entire idea of SEB absolute, so thanks for clarifying this.

**Reply by mcvanish** [reply]
Hi DBeuchel,

Thanks for commenting on my colleague. I have an additional question. SEB is supported by a community. But how is support on SEB arranged? Let's say we as an organisation encounter a security issue, who can we rely on to have this fixed immediately? Or is there a company who can support us?

Cheers, Menno Vermeulen - Business Analyst

**Reply by dbuechel** [reply]
We fix urgent security issues as soon as we get aware of them, though we'll naturally always evaluate the severity of an issue ourselves and can't always immediately follow the needs of every institution / organization which is using SEB. Members of the [SEB Alliance](https://safeexambrowser.org/alliance/members.html) however enjoy some benefits in this regard when compared to the general public.
